nkinsaul/tunes-and-brews
# Feature/brewery list

1. What's this PR do?
    - This will add a scrolling box next to the event details on the event view page that shows a list of breweries that are in the same zip code as the event venue.
1. Where should the reviewer start?
    - EventView.js
    - Breweries.js
1. How should this be manually tested?
    - Run `npm start` to run the server
    - Make sure to start on `/events` in the browser
    - Click on one of the events
1. Any background context you want to provide?
1. What are the relevant tickets?
1. Screenshots (if appropriate)
1. Questions:
